src/Drupal/DrupalExtension/Context/RawDrupalContext.php

```diff
@@ -309,10 +309,26 @@ public function logout() {
    */
   public function loggedIn() {
     $session = $this->getSession();
+    $page = $session->getPage();
+
+    // Look for the logged-in class on the body tag.
+    // This should work with almost any theme.
+    $body = $page->find('css', 'body');
+    if ($body->hasClass('logged-in')) {
```

Comment:
We should make the `logged-in` class configurable, the same way the log out text is (see `getDrupalText()` usage below for example). (Same goes for the below `form#user-login`.)